Examples/VoiceMemos/VoiceMemos/VoiceMemo.swift

```diff
@@ -112,42 +108,40 @@ struct VoiceMemoView: View {
   }
 
   var body: some View {
-    GeometryReader { proxy in
-      ZStack(alignment: .leading) {
-        if self.viewStore.mode.isPlaying {
-          Rectangle()
-            .foregroundColor(Color(.systemGray5))
-            .frame(width: proxy.size.width * CGFloat(self.viewStore.mode.progress ?? 0))
-            .animation(.linear(duration: 0.5), value: self.viewStore.mode.progress)
```

Comment:
Since we don't support pausing, I simplified and improved the visuals of the progress animation (moved to the background view modifier, below).